styles/partials/components/buttons.scss

```
@@ -0,0 +1,27 @@
+.regular-button {
+  font-size: 1rem;
+  font-weight: 600;
+  text-transform: uppercase;
+  letter-spacing: 2.5px;
+  color: $font-color-light;
+  width: 245px;
+  height: 64px;
+  border: 1px solid;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &:hover {
+    color: $font-color-dark;
+    background-color: $font-color-light;
```

Consider using a more descriptive variable name for the background color. Don't forget to add it to the variables as well
```suggestion
    background-color: $bg-color-light;
```

---

I changed it